File: PythonFiles/uberP.py
from uberCar import uberCar

class uberP(uberCar):
    brand = str
    model = str

    def __init__(self, license, driver, passengers, brand, model):
        uberCar.__init__(self,license,driver,passengers)
        self.brand = brand
        self.model = model
        def printDataUberP():
            print("-------------")
            print("Esto es uberP")
            print(self.driver.name)
            print(self.driver.email)
            print(self.driver.documentID)
            print(self.license)
            print(self.passengers)
            print(self.brand)
            print(self.model)
          
        printDataUberP()

File: PhpFiles/uberP.php
<?php
require_once('uberCar.php');
class uberP extends uberCar 
{
    public $brand;
    public $model;
    
    public function __construct($license,$driver,$passengers,$brand,$model)
    {
        parent::__construct($license,$driver,$passengers);
        $this->$brand = $brand;
        $this->$model = $model;
    }

}

?>